rootfs/etc/imgserv/imgserv_config.py

```diff
@@ -0,0 +1,53 @@
+# This file is part of dax_imgserv.
+#
+# Developed for the LSST Data Management System.
+# This product includes software developed by the LSST Project
+# (http://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+import os
+from configparser import RawConfigParser
+
+config_json = dict({
+  "DEBUG": True,
+  "DAX_IMG_TEMPDIR": "/tmp/imageworker_results",
+  "DAX_IMG_VALIDATE": True,
+  "DAX_IMG_DS": "/datasets/sdss/preprocessed/dr7/sdss_stripe82_01",
+  "DAX_IMG_TAB_SCICCDEXP": "Science_Ccd_Exposure",
+  "DAX_IMG_TAB_DEEPCOADD": "DeepCoadd",
+  "DAX_IMG_DR": "/datasets/sdss/preprocessed/dr7/runs",
+  "DAX_IMG_BUTLER_KEYS1": ["run", "camcol", "field", "filter"],
+  "DAX_IMG_COLUMNS1": ["run", "camcol", "field", "filterName"],
+  "DAX_IMG_BUTLER_KEYS2": ["tract", "patch", "filter"],
+  "DAX_IMG_COLUMNS2": ["tract", "patch", "filterName"],
+  "DAX_IMG_BUTLER_POL0": "fpC",
+  "DAX_IMG_BUTLER_POL1": "calexp",
+  "DAX_IMG_BUTLER_POL2": "deepCoadd",
+  "DAX_IMG_SKYMAP_DEEPCOADD": "deepCoadd_skyMap"
+})
```

Comment:
I'm confused by this.  This config_json seems to be the config you use elsewhere, but it's all hardcoded?

Reply:
In Python, .py file can be used as editable config file, so seeing hard-coded strings shouldn't be a surprise at all.